cpg-library/src/main/java/de/fraunhofer/aisec/cpg/graph/statements/expressions/ConstructExpression.kt

```diff
@@ -98,7 +88,7 @@ class ConstructExpression : CallExpression(), HasType.TypeListener {
         if (!TypeManager.isTypeSystemActive()) {
             return
         }
-        val subTypes: MutableSet<Type> = HashSet(getPossibleSubTypes())
+        /*val subTypes: MutableSet<Type> = HashSet(getPossibleSubTypes())
```

Comment:
Do we want to remove this commented out code or is there a readon for keeping it?

Reply:
It was intentionally left for the reviewer ;) Nah, it was left over code. I will remote it.

Reply:
So i passed the reviewing the reviewer test ;)